src/Core/src/Handlers/SwipeView/SwipeViewHandler.Android.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class SwipeViewHandler : ViewHandler<ISwipeView, MauiSwipeView>
+	{
+		protected override MauiSwipeView CreateNativeView()
+		{
+			var returnValue = new MauiSwipeView(Context)
+			{
+				CrossPlatformMeasure = VirtualView.CrossPlatformMeasure,
+				CrossPlatformArrange = VirtualView.CrossPlatformArrange
+			};
+
+			returnValue.SetElement(VirtualView);
+			return returnValue;
+		}
+
+		public override void SetVirtualView(IView view)
+		{
+			base.SetVirtualView(view);
+			_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
+			_ = NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
+
+			NativeView.CrossPlatformMeasure = VirtualView.CrossPlatformMeasure;
+			NativeView.CrossPlatformArrange = VirtualView.CrossPlatformArrange;
+		}
+
+		public static void MapContent(ISwipeViewHandler handler, ISwipeView swipeView)
+		{
+			_ = handler.NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
+			_ = handler.MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} should have been set by base class.");
+			_ = handler.TypedVirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
+
+			handler.TypedNativeView.UpdateContent();
+		}
+
+		public static void MapIsEnabled(ISwipeViewHandler handler, ISwipeView swipeView)
+		{
+			handler.TypedNativeView.UpdateIsSwipeEnabled(swipeView.IsEnabled);
+			ViewHandler.MapIsEnabled(handler, swipeView);
+		}
+
+		public static void MapBackground(ISwipeViewHandler handler, ISwipeView swipeView)
+		{
+			// TODO JAVIER is this right? I wasn't sure exactly how to combine the two
```

Comment:
@jsuarezruiz  ?

Reply:
The logic in Xamarin.Forms was, apply the brush from Background and if is null, check and apply the color from the BackgroundColor property.